app/controllers/payments_controller.rb

```diff
@@ -24,8 +25,15 @@ def create
 
     if @payment.purchase && @payment.save
       update_purchased_ticket_purchases
-      redirect_to conference_physical_tickets_path,
-                  notice: 'Thanks! Your ticket is booked successfully.'
+
+      has_registration_ticket = params[:has_registration_ticket]
+      if has_registration_ticket == 'true'
+        redirect_to new_conference_conference_registration_path(@conference.short_title),
```

Comment:
so a registration ticket means you have to register not that you are already registered?